src/pkg/packager/interactive.go

```diff
@@ -44,7 +45,25 @@ func (p *Packager) confirmAction(userMessage string, sbomViewFiles []string) (co
 
 	// Prompt the user for confirmation, on abort return false
 	if err := survey.AskOne(prompt, &confirm); err != nil {
-		return false
+		confirm = true
+	}
+
+	// On create in interactive mode, prompt for max package size
+	if userMessage == "Create" {
+		value, _ := p.promptVariable(types.ZarfPackageVariable{
```

Comment:
We should not dogsled the error here.  An interrupt (Ctrl-C) should stop Zarf as soon as possible.